### ISSUE TITLE
It just suggested a version downgrade to me

### ISSUE THREAD

**Issue by papb** [yeoman/update-notifier#181]
Screenshot:

![image](https://user-images.githubusercontent.com/20914054/84606476-ecc91400-ae7c-11ea-86fe-804eb758a8ef.png)

As text:

```
$ npx xo --fix


   ╭───────────────────────────────────────╮
   │                                       │
   │   Update available 0.32.0 → 0.30.0    │
   │        Run npm i xo to update         │
   │                                       │
   ╰───────────────────────────────────────╯

```

What is happening? :sweat_smile:

**Comment by yang** [comment]
We also see this:

![image](https://user-images.githubusercontent.com/7129/90599605-00109c80-e1aa-11ea-9a01-a8f8a870d034.png)

```
   ╭───────────────────────────────────────────────────╮
   │                                                   │
   │         Update available 0.1.53 → 0.1.51          │
   │   Run yarn global add @plasmicapp/cli to update   │
   │                                                   │
   ╰───────────────────────────────────────────────────╯
```

@sindresorhus Perhaps due to this line comparing by equality?

https://github.com/yeoman/update-notifier/blob/c9d2166b8303eba2304eff4a007a6a7fa50b6c77/index.js#L117

**Comment by sindresorhus** [comment]
> Perhaps due to this line comparing by equality?

I don't think that's the underlying reason for it, but it seems like a good workaround for the issue. Just use https://github.com/npm/node-semver to ensure the version is newer.
